deploy/base/serviceaccount.yaml

```diff
@@ -0,0 +1,6 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: grafana-operator-controller-manager
+  namespace: default
```

Comment:
probably better to remove namespace or use a better default value

Reply:
@NissesSenap this is required by kustomize in order to replace the namespace. It will apply whichever namespace is set in `base/kustomization.yaml`, but for some resources (role, rolebinding) the namespace *must* be set to default, otherwise it is not replaced. So I thought to just use default everywhere.